packages/stryker-javascript-mutator/src/JavaScriptMutator.ts

```diff
@@ -7,21 +7,23 @@ import BabelParser from './helpers/BabelParser';
 import copy from './helpers/copy';
 import NodeMutatorFactory from './NodeMutatorFactory';
 import NodeMutator from './mutators/NodeMutator';
+import * as path from 'path';
 
 function defaultMutators(): NodeMutator[] {
   return NodeMutatorFactory.instance().knownNames().map(name => NodeMutatorFactory.instance().create(name, undefined));
 }
 
 export default class JavaScriptMutator implements Mutator {
   private log = getLogger(JavaScriptMutator.name);
+  private knownExtensions: Array<string> = ['.js', '.jsx'];
```

Comment:
We can move this list to the hasValidExtension function since we only use it there

Reply:
Could you also change it to a `string[]` or remove the type entirely?